src/main/groovy/com/avioconsulting/mule/linter/model/pom/MunitMavenPlugin.groovy

```diff
@@ -0,0 +1,79 @@
+package com.avioconsulting.mule.linter.model.pom
+
+import com.avioconsulting.mule.linter.parser.MuleXmlParser
+import groovy.xml.slurpersupport.GPathResult
+
+class MunitMavenPlugin extends PomPlugin {
+
+    static final String GROUP_ID = 'com.mulesoft.munit.tools'
+    static final String ARTIFACT_ID = 'munit-maven-plugin'
+
+    MunitMavenPlugin(GPathResult pluginXml, PomFile pomFile) {
+        super(pluginXml, pomFile)
+    }
+
+    @SuppressWarnings('UnnecessaryCollectCall')
+    Map<String, String> getConfigurationCoverageDetails() {
+        Map<String, String> m = [:]
+        pluginXml.configuration.coverage.childNodes().each {
+            String value = (it.childNodes().size() > 0) ? it.childNodes().collect { it.text() }.join(',') : it.text()
+            m.put(it.name as String, value)
+        }
+        return m
+    }
+
+    String getCoverageValue(String property) {
+        pluginXml.configuration.coverage.find { it.name() == property }?.text()
+    }
+
+    List<String> getIgnoreFiles() {
+        List<String> ignoreFiles = []
+        pluginXml.configuration.coverage.ignoreFiles.ignoreFile.each {
+            ignoreFiles.add(it.text())
+        }
+        return ignoreFiles
+    }
+
+    Integer getIgnoreFilesLineNo() {
+        MuleXmlParser.getNodeLineNumber(pluginXml.configuration.coverage.ignoreFiles)
+    }
+
+}
+
+/*
+			<plugin>
```

Comment:
If we want to keep a sample in here put it in a groovydoc on the class

Reply:
Done